### DISCUSSION TITLE
SwiftUI LottieView how to use JSON that imported to project via folder references.

### DISCUSSION THREAD

**Discussion by lbininhbl** [airbnb/lottie-ios#2387] [Q&A] [answered]
In my case, the structure of all my Lottie JSON file is like this:
- animation1
- - data.json
- - - images
- - - - img_0.png

- animation2
- - data.json
- - - images
- - - - img_0.png
- - - - img_1.png

So I add all these files to project via Create folder references in UIKit, and load it using file path and it work properly.

However, I use SwiftUI LottieView in the same way and do not behave abnormally. Console shows images not found.
![iShot_2024-04-23_20 01 17](https://github.com/airbnb/lottie-ios/assets/5364138/fb7a600e-8b5c-431a-a339-494c9942d240)

Is there any best practice?

**Comment by calda** [comment, marked as answer]
You need to provide an `AnimationImageProvider` using `.imageProvider(...)`.
